vcd_cli/firewall_rule.py

```diff
@@ -79,3 +79,19 @@ def create_firewall_rule(ctx, gateway_name, name, action, type,
         stdout('Firewall rule created successfully.', ctx)
     except Exception as e:
         stderr(e, ctx)
+
+@firewall.command('list', short_help='displays all firewall rules.')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+def rules_list(ctx, name):
+    try:
+
+        gateway_resource = get_gateway(ctx, name)
+        firewall_rules = gateway_resource.get_firewall_rules()
+        firewall_rule_list = []
+        for firewall_rule in firewall_rules.iter('firewallRule'):
+            firewall_rule_list.append(dict(ID=firewall_rule.find('id').text, name=firewall_rule.find('name').text,
```

Comment:
No need to for this logic, u can directly firewall_rules  in stdout function, refer dhcp_pool.py or gateway.py for the same